app/javascript/packs/puerto-rico-guidance.ts

```diff
@@ -13,5 +17,21 @@ function onStateSelectionChange() {
   showOrHidePuertoRicoExtras(stateSelector?.value);
 }
 
+function onIdentityDocStateSelection() {
+  const stateSelectors = document.querySelectorAll<HTMLSelectElement>('.usa-select');
+  stateSelectors.forEach((stateSelector) => {
+    if (!stateSelector.id.includes('jurisdiction')) {
```

Comment:
This is less type safe than checking the element type directly.

```suggestion
  const stateSelectors = document.querySelectorAll<unknown>('.usa-select');
  stateSelectors.forEach((stateSelector) => {
    if (stateSelector instanceof HTMLSelectElement && !stateSelector.id.includes('jurisdiction')) {
```